README.md

```diff
@@ -1 +1,48 @@
-# Portfolio
\ No newline at end of file
+
+# Microverse Portfolio Project
+> Portfolio project created using Figma design.
+### Mobile View
+![screenshot](mobile.png)
+### Desktop View
+![screenshot](desktop.png)
+
+
+## Built With
+- Html
+- CSS
+- GITHub
+
+## Getting Started
+ To get a local copy up and running follow these simple example steps.
+- Clone this repository or download the Zip folder:
+git clone https://github.com/mr-anuj/Portfolio.git
+- Navigate to the location of the folder in your machine
+
+### Prerequisites
+- Web browser (like Google Chrome, Opera...).
+- Code editor (like Atom, VScode...).
+
+## Setup
+To setup the project locally: click on "Code" (dropdown list) then "Download ZIP".
+
+## Deployment
+To run the project open the downloaded folder (after extraction) then run the "index.html" file in your browser.
+
+## Authors
+
+👤 **Anuj Chettri**
+
+- GitHub: [@githubhandle](https://github.com/mr-anuj)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [OPTIONAL] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```